code/modules/clothing/glasses/hud.dm

```diff
@@ -133,4 +134,28 @@
 	item_state = "glasses"
 	darkness_view = 8
 	invis_view = SEE_INVISIBLE_MINIMUM //don't render darkness while wearing these
-	prescription_upgradable = 0
\ No newline at end of file
+	prescription_upgradable = 0
+
+/obj/item/clothing/glasses/hud/security/tajblind
+	name = "sleek veil"
+	desc = "An Ahdominian made veil that allows the user to see while obscuring their eyes. This one has an in-built security HUD."

```

Comment:
"This one has a security HUD built-in." or "This one has a built-in security HUD." sound more natural.